components/console-backend-service/internal/filter/by_labels.go

```diff
@@ -0,0 +1,53 @@
+package filter
+
+import (
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/api/meta"
+)
+
+func ByLabels(items []interface{}, labels []string) ([]interface{}, error) {
```

Comment:
When I first read the code, I expected this function to filter services that contain those labels, but it turned out to be the exact opposite. Could you please rename it so it's clear what this function does?

Reply:
I added `excludeByLabels` and `includeByLabels` functions :)